lib/authentication/firebase_authentication.dart

```diff
@@ -55,6 +55,10 @@ class Auth implements BaseAuth {
     user.sendEmailVerification();
   }
 
+  Future<void> resetPassword(String email) async {
```

Comment:
Originally did it like this:
```dart
@override
Future<void> sendPasswordResetEmail(email: email) async {
     await _firebaseAuth.sendPasswordResetEmail(email: email);
}
```

But it said that it couldn't find the firebase method to override. Therefore to avoid confusion, I just named it `resetPassword` and made the `sendPasswordResetEmail` call within.

Reply:
Sounds good! I was gonna ask about that!